scripts/linux/runE2ETest.sh

```diff
@@ -187,6 +187,10 @@ function prepare_test_from_artifacts() {
         sed -i -e "s/<Build.BuildNumber>/$ARTIFACT_IMAGE_BUILD_NUMBER/g" "$deployment_working_file"
         sed -i -e "s@<CR.Username>@$CONTAINER_REGISTRY_USERNAME@g" "$deployment_working_file"
         sed -i -e "s@<CR.Password>@$CONTAINER_REGISTRY_PASSWORD@g" "$deployment_working_file"
+
+        if [[ $CONTAINER_REGISTRY != 'edgebuilds.azurecr.io' ]]; then
```

Comment:
I would suggest to update our deployment files to have a new tag <Container_Registry> and replace it by default value "edgebuilds.azurecr.io".  You can either fix this in this PR or create a new PR.